### ISSUE TITLE
SC 2.4.11 Focus Not Obscured (Minimum) - User openable, persistent disclosures - WCAG standard example

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#3825]
The second example of the "User openable, persistent disclosures" scenario states:

> When the additional content appears, existing content reflows. The popout sidebar on the [WCAG standard](https://www.w3.org/TR/WCAG22/) is an example of this pattern. When the side menu is activated, it opens a new section of information along the left side of the page. The main content area is reduced horizontally to accommodate the new content, and the existing content reflows to fit in the thinner space. As a result, there is no overlapping content between the two sections; the item receiving focus, whether in the left navigation or in the main content, will not be obscured by the other section.

However, this WCAG website behavior is only consistent on desktop with a 100% zoom level. 
On mobile devices and desktops with different zoom levels (such as 200%), this alignment is no longer valid. 

Additionally, on mobile devices (and also with a 200% zoom on desktop), the success criterion itself fails as elements receiving focus become obscured by the sidebar menu and users are unable to reveal the focused component without advancing the keyboard focus.
